src/app/assets/languages/en.json

```diff
@@ -394,7 +394,7 @@
           "recovery-cache": "Recovery cache",
           "recovery-cache-tooltip": "Specifies the cache that stores information to recover in-doubt transactions.",
           "transactional-tuning": "Transaction tuning",
-          "transactional-tuning-tooltip": "Configures settings for transactional caches."
+          "transactional-tuning-tooltip": "Configure and adjust settings for transactional caches."
```

Comment:
I think either "Configure settings" or "Adjust settings" is good. Maybe "adjust" because it's more of a synonym with "tune".